en/activity-standard/overview/country-budget-alignment.rst

```diff
@@ -0,0 +1,40 @@
+Country Budget Alignment
+=========
+
+Guidance on implementing budget and aid alignment
+----------
+During the 2.03 upgrade process, decision was made to deprecate BudgetIdentifier codelist. As a result, further guidance was required to provide to publishers on implementing aid and budget alignment.
```

Comment:
As a minimum I would recommend that the narrative of [`BudgetIdentifier`](https://github.com/IATI/IATI-Codelists-NonEmbedded/blob/master/xml/BudgetIdentifier.xml#L7) gets updated to indicate that it is deprecated at version 2.03. Going forwards, the technical process for how a non-embedded codelist is removed should also be clarified.

Reply:
> As a minimum I would recommend that the narrative of [BudgetIdentifier](https://github.com/IATI/IATI-Codelists-NonEmbedded/blob/master/xml/BudgetIdentifier.xml#L7) gets updated to indicate that it is deprecated at version 2.03.

I think removing references to it is even-more-minimum than that. There’s a pull request for that here:
https://github.com/IATI/IATI-Codelists/pull/139

Reply:
> Going forwards, the technical process for how a non-embedded codelist is removed should also be clarified.

Does [the Vocabulary codelist](http://iatistandard.org/203/codelists/Vocabulary/) provide a model for this? I.e. remove at the next integer, leaving an explanatory note in its place.